plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogConnectorSmokeTest.java

```diff
@@ -51,13 +60,25 @@
 {
     private final String bucketName;
     private final String schemaName;
+    private final String catalog;
+    private final AWSGlueAsync glueClient;
 
     @Parameters("s3.bucket")
     public TestIcebergGlueCatalogConnectorSmokeTest(String bucketName)
     {
         super(FileFormat.PARQUET);
         this.bucketName = requireNonNull(bucketName, "bucketName is null");
         this.schemaName = "test_iceberg_smoke_" + randomTableSuffix();
+        this.catalog = "iceberg";
```

Comment:
Not needed  because it is used only once. 

You could use a constant instead of a instance field.

Reply:
Removed and added as constant